### ISSUE TITLE
Improve speed of mybinder build/start

### ISSUE THREAD

**Issue by rymurr** [projectnessie/nessie#1984]

**Comment by rymurr** [comment]
https://github.com/jupyterhub/repo2docker-action < we should be able to build the image and publish it to docker?

**Comment by omarsmak** [comment]
> https://github.com/jupyterhub/repo2docker-action < we should be able to build the image and publish it to docker?

We can as well have that plus the prepared `Flink, Spark, Hive .. etc` artifactories  as Docker image. It will be a bit larger but I guess better than waiting for the build to complete?

**Comment by rymurr** [comment]
> We can as well have that plus the prepared `Flink, Spark, Hive .. etc` artifactories as Docker image

Yes, this is why it takes so long right now. It has to download and prepare each engine when the image starts.
